packages/dds/cell/src/cell.ts

```diff
@@ -287,7 +287,7 @@ export class SharedCell<T extends Serializable = any> extends SharedObject<IShar
             if (local) {
                 const messageIdReceived = localOpMetadata as number;
                 assert(messageIdReceived !== undefined && messageIdReceived <= this.messageId,
-                    "messageId is incorrect from from the local client's ACK");
+                    "s_00" /* messageId is incorrect from from the local client's ACK */);
```

Comment:
Are the checked in version be short code?

Reply:
i planned to commit them to keep them unique and constant without needing a ton of tracking, currently the tagging script is stateless